src/dataCommand.ts

```diff
@@ -182,7 +182,11 @@ export abstract class DataCommand extends SfdxCommand {
         throw new Error(messages.getMessage('TextUtilMalformedKeyValuePair', [pair]));
       } else {
         const key = pair.substr(0, eqPosition);
-        constructedObject[key] = this.convertToBooleanIfApplicable(pair.substr(eqPosition + 1));
+        if (pair.includes('{') && pair.includes('}')) {
```

Comment:
I could see this being true but not being parseable JSON (ex: a big textArea that's referring to some code like a Case).

So it's probably a fine solution but we'll need a try/catch on the JSON.parse because it might not be, and then we'd want to return the original value?